darling/src/libsystem_kernel/emulation/linux/misc/sysctl_kern.c

```diff
@@ -19,6 +19,7 @@
 enum {
 	_KERN_MSGBUF = 1000,
 	_KERN_OSPRODUCTVERSION,
+	_KERN_OSREVISION,
```

Comment:
This is the wrong value; the actual value is defined in `sys/sysctl.h` (which is already included in this file). You can find the right value in the `KERN_OSREV` macro.

```suggestion
```